lib/daru/maths/statistics/vector.rb

```diff
@@ -7,6 +7,8 @@ module Statistics
       module Vector
         def mean
           @data.mean
+          rescue ZeroDivisionError
```

Comment:
A more robust way of handling this is to change the wrapper methods that handle basic stats operations like mean and product. See this: https://github.com/v0dro/daru/blob/master/lib/daru/accessors/array_wrapper.rb#L70

You can make it return a nil if the size of `@context.missing_positions` is `0`.